util.c

```diff
@@ -61,6 +61,7 @@ uint64_t mstime(void)
 {
     struct timeval tv;
     uint64_t mst;
+	int   gettimeofday(struct timeval *, void *);
```

Comment:
Include the correct header for this, don't prototype it yourself.

Where does bsd put gettimeofday, if not in sys/time.h?